apps/native/app/src/screens/health/health-overview.tsx

```diff
@@ -376,6 +381,70 @@ export const HealthOverviewScreen: NavigationFunctionComponent = ({
               noBorder
             />
           </InputRow>
+          <HeadingSection
+            title={intl.formatMessage({
+              id: 'health.overview.medicinePurchase',
+            })}
+            onPress={() =>
+              openBrowser(
+                `${origin}/minarsidur/heilsa/lyf/lyfjakaup`,
+                componentId,
+              )
+            }
+          />
+          <InputRow background>
+            <Input
+              label={intl.formatMessage({
+                id: 'health.overview.period',
+              })}
+              value={
+                medicinePurchaseRes.data?.rightsPortalDrugPeriods?.[0]
```

Comment:
nit: store `medicinePurchaseRes.data?.rightsPortalDrugPeriods?.[0]` item in a variable since it used 8 times in JSX